.travis.yml

```diff
@@ -2,6 +2,7 @@ language: ruby
 
 before_install:
   - gem install bundler -v '< 2'
+  - "find . -type f -iname '*.rb' -exec sed -i '1s/^/# frozen_string_literal: true\\n\\n/' \"{}\" +;"
```

Comment:
This is clever - though we actually want to add Rubocop to our repo and it would be beneficial to put this on every file. Is that something you'd be willing to try?

Reply:
The original theme of this pull request is "least intrusive".
But if you are OK, after I finished playing with Travis, I can:
- commit what `sed` changes
- remove all the hacks
- Find a way to enable `Style/FrozenStringLiteralComment` and let rubocop do the jobs.

How does it sound?

Reply:
Sounds fine with me! I anticipate a lot of effort, so be aware... this is a big repo.